src/dom-updates.js

```diff
@@ -0,0 +1,116 @@
+const domUpdates = {
+    // CHANGE THE DISPLAY OF ANY ARGS PASSED IN, MUST PASS IN [ARGS]
+    toggle(anyArr) {
+        anyArr.forEach(area => 
+            document.querySelector(area).classList.toggle("hidden"))
+    },
+
+    // HELPER FUNCTION TO INSERTHTML
+    addDisplay(area, where, what) {
+        document.querySelector(area).insertAdjacentHTML(where, what)
+    },
+
+    clearDisplay(area) {
+      document.querySelector(area).innerHTML = ''
+    },
```

Comment:
This is very smart, and I love the reusability/flexibility you have built into this very useful function!